trinity/common/config.py

```diff
@@ -201,8 +200,6 @@ class MonitorConfig:
     # TODO: add more
     project: str = "trinity"
     name: str = "rft"
-    group: str = ""
```

Comment:
Remove the `group`  field in `tests/common/tmp/template_config.yaml`

Reply:
OK